etc/linux-systemd/user/syncthing.service

```diff
@@ -1,12 +1,14 @@
 [Unit]
 Description=Syncthing - Open Source Continuous File Synchronization
+ConditionUser=!@system
```

Comment:
I agree running a user service as a system user doesn't make much sense, but I still don't like this condition much. The lesser reason is that I think it's superfluous (I don't think anyone does that) and more importantly, ccording to the man-page it silently skips the script, it doesn't fail it, i.e. there is no obvious feedback to the user that it failed.

Reply:
What does "system user" mean here? I (and others) often run Syncthing as a low-uid user without a password and no login rights. This is perfectly fine.